### PR TITLE
Fix for Text Mode toolbar

### PR DESCRIPTION
**Pull request by ShreyaSingh10**
## Description

The Text Mode toolbar is entirely visible in every screen resolution
## How Has This Been Tested?

With the help of Chrome Inspector
https://github.com/WordPress/gutenberg/issues/3041

## Checklist:

### CODE REVIEW

**Review by aduth — CHANGES_REQUESTED** [review]
It appears beyond just the formatting controls that the entire editor component layout (`.editor-layout__editor`) is potentially hidden behind the fixed editor toolbar on small screens. In visual mode this is offset by some padding on `.editor-visual-editor` but it doesn't seem to be enough. I suggest we add margin to the `.editor-layout__editor`.

Also be mindful of CSS coding guildelines, where the introduced styles should be structured using tabs, not spaces:

https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#structure

**Comment by ShreyaSingh10** [comment]
@aduth I have changed the margin to the css class you suggested. Though now it looks aligned slightly to left if you can confirm i will fix that.

**Comment by aduth** [comment]
I'm going to add a Design Review label because while I agree we should have this margin, it prompts whether we should then adjust the padding of `.editor-visual-editor` in smaller displays. Now there is a fairly large gap between the toolbar and title in mobile viewports in Visual editing mode.

Gap

>Though now toolbar looks aligned slightly to right if you can confirm i will fix that.

I'm not seeing this myself.

File: editor/modes/text-editor/style.scss [hunk 1]
```diff
@@ -132,3 +132,9 @@
 	max-width: $text-editor-max-width;
 	margin: 0 auto;
 }
+
+@media (max-width: 599px) {
```
Comment (aduth) [inline editor/modes/text-editor/style.scss]: I think we ought to define this style mobile-first with `margin-top: 56px;` and a media query to reset on larger displays. There are a number of [mixins](https://github.com/WordPress/gutenberg/blob/e869a9dcde45197feb9d1ebc1b45047c1da551a5/editor/assets/stylesheets/_mixins.scss#L1-L35) and [variables](https://github.com/WordPress/gutenberg/blob/e869a9dcde45197feb9d1ebc1b45047c1da551a5/editor/assets/stylesheets/_variables.scss#L111-L116) available for this. I think you'll want:

```scss
margin-top: 56px;

@include break-small {
	margin-top: 0;
}
```

...inside the `.editor-layout__editor` selector.

**Comment by ShreyaSingh10** [comment]
@aduth I have done the changes.

**Comment by aduth** [comment]
@jasmussen Do you have any thoughts on how we might best address the gap mentioned in https://github.com/WordPress/gutenberg/pull/3043#issuecomment-338306500 ?

**Comment by jasmussen** [comment]
Thank you for the PR, and thank you for opening #3041, appreciate it. The ticket certainly does reveal that we haven't given the text mode the love it deserves.

While there are issues in master, I don't think there's an easy fix, and although I appreciate the work done here, I suspect that we'll have to go a bit back to the drawing board when it comes to text mode. There's even disagreement whether the quick tags toolbar should even be there, and given the incoming codemirror implementation in 4.9, I suspect we'll want to redo that section regardless. As such, I would thank you for this PR, but I would not merge it, or even try to fix the implementation as it is today. Since I'm the person who built text mode as it is today, sorry about that ;)

Instead, I will open a new ticket to redesign the text mode once codemirror hits 4.9, and I will link your ticket as well as other past text mode tickets, so we can have a new umbrella ticket for improvements. 

Some of the issues highlighted by this PR can be seen here:

screen shot 2017-10-24 at 09 14 09

screen shot 2017-10-24 at 09 14 13

screen shot 2017-10-24 at 09 14 16

screen shot 2017-10-24 at 09 14 26

screen shot 2017-10-24 at 09 14 30

screen shot 2017-10-24 at 09 14 33

**Comment by gziolo** [comment]
@ShreyaSingh10 do you plan to finish this PR or should someone else take it over?

**Comment by youknowriad** [comment]
Closing as this toolbar has been removed for now. Thanks for the time spent on this PR.

**Pull request closed without merging**